src/structures/GuildChannel.js

```diff
@@ -213,11 +213,19 @@ class GuildChannel extends Channel {
     return this;
   }
 
+  /**
+   * Extra information about the overwrite
+   * @typedef {Object} ExtraOverwriteInfo
+   * @property {string} [reason] Reason for creating/editing this overwrite
+   * @property {string} [type] The type of overwrite, either `member` or `role`. Use this to bypass automatic
```

Comment:
While a string still seems to work (for now), [the documentation specifies an integer](https://discord.com/developers/docs/resources/channel#edit-channel-permissions).

```suggestion
   * @property {number} [type] The type of overwrite, either `0` for a role or `1` for a member. Use this to bypass automatic
```